docs/user/APIServices.md

```diff
@@ -7,9 +7,9 @@ Make sure your Google Cloud Project has at least the following API Services enab
 | `container.googleapis.com` | [Kubernetes Engine API](https://console.cloud.google.com/apis/library/container.googleapis.com) ||
 | `compute.googleapis.com`| [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com) ||
 | `file.googleapis.com` | [Cloud Filestore API](https://console.cloud.google.com/apis/library/file.googleapis.com) | Needed for `storage_type="ha"` |
-| `sqladmin.googleapis.com`| [Cloud SQL Admin API](https://console.cloud.google.com/apis/library/sqladmin.googleapis.com) | Needed when creating an [SQL Postgres instance](../CONFIG-VARS.md#postgres)
-| `servicenetworking.googleapis.com`| [Service Networking API](https://console.cloud.google.com/apis/library/servicenetworking.googleapis.com) | Needed when creating an [SQL Postgres instance](../CONFIG-VARS.md#postgres)
-| `cloudresourcemanager.googleapis.com`| [Cloud Resource Manager API](https://console.cloud.google.com/apis/library/cloudresourcemanager.googleapis.com) | Needed if you create an [SQL Postgres instance](../CONFIG-VARS.md#postgres) |
+| `sqladmin.googleapis.com`| [Cloud SQL Admin API](https://console.cloud.google.com/apis/library/sqladmin.googleapis.com) | Needed when creating an [SQL Postgres instance](../CONFIG-VARS.md#postgres-servers) |
```

Comment:
Is this to add clarity? Again asking as these changes seem a bit trivial.

Reply:
Minor change, just updated the link from ../CONFIG-VARS.md#postgres to ../CONFIG-VARS.md#postgres-servers

Correcting the link to "postgres-servers" will make the page auto scroll to the section when CONFIG-VARS.md gets opened.